django_slack/__init__.py

```diff
@@ -262,3 +262,8 @@
 """
 
 from .api import slack_message
+from .app_settings import app_settings
+
+# auto register our celery task before startup when we use CeleryBackend
+if 'CeleryBackend' in app_settings.BACKEND:
```

Comment:
I really don't like this I'm afraid. If we want backend-specific behaviour we should surely do it on the backend class in some way, not based on completely random strings.

(This also breaks ``override_settings``, but that's not my main objection.)